src/queries/__tests__/lpFills.test.ts

```diff
@@ -113,4 +114,59 @@ describe('getLpFills', () => {
       ]
     `);
   });
+
+  it('should correctly sort lpFills by filledAmountValueUsd in descennding order', async () => {
+    vi.spyOn(lpClient, 'request').mockResolvedValueOnce({
+      limitOrders: {
+        groupedAggregates: [
+          {
+            keys: ['1'],
+            sum: {
+              filledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['2'],
+            sum: {
+              filledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['3'],
+            sum: {
+              filledAmountValueUsd: '3',
+            },
+          },
+        ],
+      },
+      rangeOrders: {
+        groupedAggregates: [
+          {
+            keys: ['2'],
+            sum: {
+              baseFilledAmountValueUsd: '1',
+              quoteFilledAmountValueUsd: '1',
+            },
+          },
+          {
+            keys: ['3'],
+            sum: {
+              baseFilledAmountValueUsd: '0.5',
+              quoteFilledAmountValueUsd: '0.5',
+            },
+          },
+        ],
+      },
+    });
+
+    const start = '2024-10-31T00:00:00Z';
+    const end = addWeeks(start, 1).toISOString();
+    const result = await getLpFills({ start, end });
+
+    const sorted = [...result].sort((a, b) =>
+      b.filledAmountValueUsd.comparedTo(a.filledAmountValueUsd),
+    );
+
+    expect(sorted).toEqual(result);
```

Comment:
otherwise the error messages might be a bit confusing
```suggestion
    expect(result).toEqual(sorted);
```

Reply:
we could also use `expect(sorted).toMatchInlineSnapshot(` and hardcode the expected result (like in the test above). no strong opinion 🙂

Reply:
Changed, thank you ☺️